tasks/conf.yml

```diff
@@ -47,6 +47,11 @@
   when: marathon_mesos_role != ""
   notify: Restart marathon
 
+- name: Set optional --logging_level option
+  template: src=logging_level.j2 dest=/etc/marathon/conf/logging_level
+  when: marathon_logging_level != "" or marathon_logging_level != "info"
```

Comment:
`marathon_logging_level != "info"` this brings cofusion to one who will set variable to `info` level, maybe just leave first part?

Reply:
You're right. I've updated it.